doc/ale-handlebars.txt

```diff
@@ -6,9 +6,8 @@ ALE Handlebars Integration                             *ale-handlebars-options*
 ember-template-lint                          *ale-handlebars-embertemplatelint*
 
 g:ale_handlebars_embertemplatelint_executable
-                              \ *g:ale_handlebars_embertemplatelint_executable*
-
-  Type: |String|
+                                *g:ale_handlebars_embertemplatelint_executable*
+  Type: |String|                  *b:ale_handlebars_embertemplatelint_executable*
```

Comment:
This one was a bit weird to format but I've done it like in other places where there is a tag for `g:` and `b:` variables: One line between the variable name and the beginning of Type / Default.

Like this one: https://github.com/adriaanzon/ale/blob/7230cbe9e43d64a02212ae4803abff874e39b8d6/doc/ale.txt#L366